src/testbase.rs

```diff
@@ -37,6 +39,21 @@ impl Bounded for UnitBox {
     }
 }
 
+impl BHShape for UnitBox {
+    fn set_bh_node_index(&mut self, index: usize) {
+        self.node_index = index;
+        if self.has_good_bh_position {
+            panic!("An UnitBox's BVHNode has been moved even though it shouldn't have.");
```

Comment:
We need some structure which sopports moving, for future tests. Probably you can then just remove this check here.